services/api/README.md

```diff
@@ -65,15 +65,15 @@ All configuration is done using environment variables. The default values in `.e
 - `APP_NAME` - Default product name to be used in emails `Bedrock`
 - `APP_URL` - URL for app defaults to `http://localhost:2200`
 - `POSTMARK_FROM` - Reply email address `no-reply@bedrock.foundation`
-- `POSTMARK_APIKEY` - APIKey for Postmark `[change me]`
+- `POSTMARK_API_KEY` - APIKey for Postmark `[change me]`
```

Comment:
can we lose the [change me] seems stupid now

Reply:
ah yeah, totally :)